packages/ripple-keypairs/src/signing-schemes/ed25519/index.ts

```diff
@@ -42,6 +42,13 @@ const ed25519: SigningMethod = {
       signature,
       new Uint8Array(message),
       publicKey.slice(2),
+      // TODO: @noble/curves sets zcash friendly defaults
```

Comment:
I don't think the todo is necessary.  The rest of the comments are helpful.

Reply:
It was for you all to double check

Reply:
sorted

Reply:
TODO: https://github.com/XRPLF/rippled/blob/e22e3b3e02dccdaf088816c83ffbe7354a21c197/src/ripple/protocol/impl/PublicKey.cpp#L156-L172

Reply:
https://github.com/paulmillr/noble-curves/blob/d0294bb2a66b72bb955f14fdbe9ca91f219cdc21/src/abstract/edwards.ts#L473
```typescript
function verify(sig: Hex, msg: Hex, publicKey: Hex, options = verifyOpts): boolean {
    const { context, zip215 } = options;
    const len = Fp.BYTES; // Verifies EdDSA signature against message and public key. RFC8032 5.1.7.
    sig = ensureBytes('signature', sig, 2 * len); // An extended group equation is checked.
    msg = ensureBytes('message', msg);
    if (prehash) msg = prehash(msg); // for ed25519ph, etc

    const s = ut.bytesToNumberLE(sig.slice(len, 2 * len));
    // zip215: true is good for consensus-critical apps and allows points < 2^256
    // zip215: false follows RFC8032 / NIST186-5 and restricts points to CURVE.p
    let A, R, SB;
    try {
      A = Point.fromHex(publicKey, zip215);
      R = Point.fromHex(sig.slice(0, len), zip215);
      SB = G.multiplyUnsafe(s); // 0 <= s < l is done inside
```

https://github.com/paulmillr/noble-curves/blob/d0294bb2a66b72bb955f14fdbe9ca91f219cdc21/src/abstract/edwards.ts#L319
```typescript
    multiplyUnsafe(scalar: bigint): Point {
      let n = assertGE0(scalar); // 0 <= scalar < CURVE.n
      if (n === _0n) return I;
      if (this.equals(I) || n === _1n) return this;
      if (this.equals(G)) return this.wNAF(n).p;
      return wnaf.unsafeLadder(this, n);
    }
```

https://github.com/paulmillr/noble-curves/blob/d0294bb2a66b72bb955f14fdbe9ca91f219cdc21/src/abstract/edwards.ts#L138
```typescript
  function assertGE0(n: bigint) {
    // n in [0..CURVE_ORDER-1]
    return n === _0n ? n : assertInRange(n, CURVE_ORDER); // GE = prime subgroup, not full group
  }
```